labs/src/main/scala/org/scalalabs/basic/lab03/OptionExercise.scala

```diff
@@ -25,7 +25,12 @@ object OptionExercise01 {
    * - does not exist: 					"not existing"
    */
   def roomState(rooms: Map[Int, Option[String]], room: Int): String = {
-    error("Fix me")
+    rooms get room match {
+      case None => "not existing"
+      case Some(None) => "empty"
+      case Some(Some("locked")) => "not available"
```

Comment:
I made this work by guess and checking and glancing at the sample solutions.  Not sure why the double `Some` is needed.